packages/create-sitecore-jss/src/templates/angular/src/app/components/styleguide-layout/styleguide-layout.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="row">
+  <div class="col-sm-8 col-lg-10">
+    <sc-placeholder name="<%- appPrefix ? `${helper.getPascalCaseName(appName)}-` : "" %>-jss-styleguide-layout" [rendering]="rendering"></sc-placeholder>
```

Comment:
If `appPrefix` is empty, here you will have `-jss-styleguide-layout`, remove `-`